test/braitenberg.py

```diff
@@ -20,6 +20,7 @@ def main():
         rayMaxDegrees=30,
         raysPerSide=(n_rays - 1) // 2,
         play=False,
+        inference=True,
```

Comment:
why was inference set true here and was not the case before the PR?

Reply:
Because presumably the user would want to watch the agent completing the task to confirm expected behaviour.